cmake/modules/RootBuildOptions.cmake

```diff
@@ -102,37 +102,37 @@ ROOT_BUILD_OPTION(cxx14 OFF "Build using C++14 compatible mode, requires gcc > 4
 #ROOT_BUILD_OPTION(cxx17 OFF "Build using C++17 compatible mode, requires gcc >= 7.1.0 or clang")
 ROOT_BUILD_OPTION(cxxmodules OFF "Compile with C++ modules enabled.")
 ROOT_BUILD_OPTION(davix ON "DavIx library for HTTP/WEBDAV access")
-ROOT_BUILD_OPTION(dcache ON "dCache support, requires libdcap from DESY")
+ROOT_BUILD_OPTION(dcache OFF "dCache support, requires libdcap from DESY")
 ROOT_BUILD_OPTION(exceptions ON "Turn on compiler exception handling capability")
 ROOT_BUILD_OPTION(explicitlink ON "Explicitly link with all dependent libraries")
 ROOT_BUILD_OPTION(fftw3 ON "Fast Fourier Transform support, requires libfftw3")
-ROOT_BUILD_OPTION(fitsio ON "Read images and data from FITS files, requires cfitsio")
+ROOT_BUILD_OPTION(fitsio OFF "Read images and data from FITS files, requires cfitsio")
 ROOT_BUILD_OPTION(fortran ON "Enable the Fortran components of ROOT")
```

Comment:
Another thing that could be turned `OFF` by default, no? Isn't this just enabling hbook support and similar old things?

Reply:
:+1:

Reply:
It is more than that.  It allows reading FITS file (astronomy files).  Why should we disable it by default?

Reply:
I'm talking about Fortran here. Do people really need that to read FITS? Also, I see no problem letting people that need FITS enable it if they need it. We can, of course, distribute ROOT with it turned ON. The changes here are meant to save regular users some compile time.

Reply:
Well ... One annoyance is changing the default is that it will now silently no longer build (what you care about) ... then the user (that needs the disable part) will be puzzled/annoyed.

Reply:
Perhaps we can say here that even if the number of people dealing with FITS every day is small, the burden to build the interface is also very little?